.vsts-ci.yml

```diff
@@ -0,0 +1,140 @@
+variables:
+- name:  Build.Repository.Clean
+  value: true
+
+trigger:
+- release/2.1*
+- release/2.2*
+
+pr:
+- release/2.1*
+- release/2.2*
+
+jobs:
+- job: debug_osx
+  displayName: OSX Debug
+  pool:
+    vmImage: macOS-10.13
+  steps:
```

Comment:
Should add this step.  I noticed that repos that don't end up with dirty enlistments at times

```yaml
      - checkout: self
        clean: true
```

Reply:
Thanks

Reply:
I have Build.Repository.Clean as a global variable in the YAML (at the top).  I think this does the same thing. Any issue with that?